scalafix-tests/input/src/main/scala/test/disableSyntax/DisableSyntaxBase.scala

```diff
@@ -52,8 +60,12 @@ case object DisableSyntaxBase {
     def -(other: String): String = s"$value - $other"
   }
 
+  5 // assert: DisableSyntax.magicNumbers
```

Comment:
Can we assert the contents of the message here?
```scala
 5 /* assert: DisableSyntax.magicNumbers
 ^
 Numbers (5 in this instance) should always ...
*/
```
See end of this section https://scalacenter.github.io/scalafix/docs/developers/tutorial.html#use-diagnostic-to-report-linter-errors